gspread/client.py

```diff
@@ -316,9 +318,9 @@ def copy(
                     continue
 
                 new_spreadsheet.share(
-                    email_address=p["emailAddress"],
-                    perm_type=p["type"],
-                    role=p["role"],
+                    email_address=str(p["emailAddress"]),
+                    perm_type=str(p["type"]),
+                    role=str(p["role"]),
```

Comment:
this is not a typing change

Reply:
it's not, the reason here is:
- the method `original.list_permissions` return a Dict wich contains values as `str` or `bool`
- mypy complains that we try to set values that could be boolean to a string (perm_type=p["type"])

In order to make it understand this is a string we should use the `str` function.